<?xml version="1.0"?>

<launch>
    <node pkg="darpa_reporting" type="darpa_reporting_node" name="darpa_reporting_node" output="screen">
        <param name="robot_frames" type="yaml" value="[base_link]" />
        <param name="darpa_frame" value="darpa" />
        <param name="map_frame" value="map" />
        <param name="map_topic" value="map" />
        <param name="map_publish_topic" value="map_relay" />
        <param name="poses_publish_topic" value="poses_relay" />
        <param name="map_publish_rate" value="0.1"/>
        <param name="poses_publish_rate" value="0.1" />
        <param name="transform_dimension" value="4" />
    </node>
</launch>
